Rakefile

```diff
@@ -1,6 +1,72 @@
 require 'bundler/gem_tasks'
 require 'rspec/core/rake_task'
+require 'mauth/request_and_response'
+require 'mauth/client'
+require 'securerandom'
+require 'benchmark/ips'
+require 'faraday'
+require "rspec/mocks/standalone"
 
 RSpec::Core::RakeTask.new(:spec)
 
 task default: :spec
+
+
+class TestSignableRequest < MAuth::Request
+  include MAuth::Signed
+  attr_accessor :headers
+  def merge_headers(headers)
+    self.class.new(@attributes_for_signing).tap{|r| r.headers = (@headers || {}).merge(headers) }
+  end
+  def x_mws_time
+    headers['X-MWS-Time']
+  end
+  def x_mws_authentication
```

Comment:
[nits] have empty lines between methods for readability? I understand it's utility class you might want to keep compact but I think the pros overweight the cons as far as new lines are concerned